api/src/main/java/org/apache/iceberg/io/FileInfo.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.io;
+
+import java.time.Instant;
+
+public class FileInfo {
+  private final String location;
+  private final long size;
+  private final Instant created;
+
+  public FileInfo(String location, long size, Instant created) {
```

Comment:
In the rest of the code, we use a plain long value in milliseconds for timestamps like this rather than using `Instant`. Can you switch to `long` or is there a reason to use `Instant` instead?

Reply:
No specific need (other than it removes confusion about the unit).